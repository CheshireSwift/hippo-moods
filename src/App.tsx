import React from 'react';
import { css } from 'emotion';

export const App = () => <div className={css({ color: 'magenta' })}>Hi</div>;

export default App;
