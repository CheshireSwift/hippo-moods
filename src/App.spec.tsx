import React from 'react';
import { render } from 'enzyme';
import App from './App';

describe('the App component', () => {
  it('matches the snapshot', () => {
    expect(render(<App />)).toMatchSnapshot();
  });
});
