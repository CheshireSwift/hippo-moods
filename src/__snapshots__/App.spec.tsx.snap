// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`the App component matches the snapshot 1`] = `
.emotion-0 {
  color: magenta;
}

<div
  class="emotion-0"
>
  Hi
</div>
`;
